pinot-common/src/main/java/com/linkedin/pinot/common/config/TagNameBuilder.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.common.config;
+
+import com.linkedin.pinot.common.utils.ServerType;
+import com.linkedin.pinot.common.utils.TenantRole;
+
+
+public class TagNameBuilder {
```

Comment:
Is this supposed to be a builder? it has all kinds of methods get, build, has. It resembles a Util class

Reply:
Agreed, Builder is not quite the right name, given there is a pattern around that name.

TagNameUtils?